packages/bee-q/src/components/breadcrumb/_storybook/bq-breadcrumb.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+
+import { html } from 'lit-html';
+
+import mdx from './bq-breadcrumb.mdx';
+
+const meta: Meta = {
+  title: 'Components/Breadcrumb',
+  component: 'bq-breadcrumb',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    text: { control: 'text', table: { disable: true } },
+    ['aria-label']: { control: 'text' },
+    htmlNode: { control: { type: 'object' } },
```

Comment:
We don't want this to be shown in the control panel, since it is not part of the component

```suggestion
    htmlNode: { control: 'object',  table: { disable: true } },
```